src/bootstrap.js

```diff
@@ -446,7 +447,10 @@ module.exports = (ceConfig = {}, pnConfig = {}) => {
 
             if (ChatEngine.me.uuid === args[0]) {
                 return ChatEngine.me;
+            } else if (ChatEngine.users[args[0]]) {
+                return ChatEngine.users[args[0]];
```

Comment:
This prevents us from recreating users we've already made, saves tons of memory. Everything is singleton.